### ISSUE TITLE
Add customizer preview support

### ISSUE THREAD

**Issue by westonruter** [Automattic/Picard#54]
WordPress 4.7 has specific improvements to the customizer to make the preview compatible with JS-driven themes. I was originally thinking I would submit a PR to Picard to add support, but the theme fatal errored for me and so I found another theme to add support to. I did so for Anadama, see https://github.com/ryelle/Anadama-React/pull/5

If Picard gets freshened up to be compatible with WordPress 4.7 and the REST API committed to core, I'd like to help work out integration with the customizer.

**Comment by westonruter** [comment]
I wasn't aware that Picard depends on v1 of the API, so that's probably why I got fatals.

**Comment by kwight** [comment]
No, Picard hasn't seen any active development for quite some time – awesome that you added it to Anadama! Thanks for noting.
